romancal/regtest/test_wfi_pipeline.py

```diff
@@ -23,7 +23,7 @@ def passfail(bool_expr):
 
 @pytest.mark.bigdata
 @pytest.mark.soctests
-@metrics_logger("DMS86", "DMS87", "DMS88")
+@metrics_logger("DMS86", "DMS87", "DMS88", "DMS280")
```

Comment:
Did we have any other DMS requirements we wanted to close as part of this?  I'm looking for the DMS362, 366, 363, 367?  Or you're waiting to plug in the data before decorating those?

Reply:
If I have read them correctly, which is possibly not, DMS362, 366, 363, and 367, all relate to just the ramp fit step, which are now all handled in the `test_ramp_fit.py` test module, which has just been pushed.

Reply:
Ah, good, I see this now.  We need to decorate the tests with the metrics_logger decoration so that whatever tools pick up the tests right.  It's not clear to me that it's straightforward with your current structure, but ideally we'd have a single DMS362: Pass, and not three of them.